app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Beer List :: East Nashville Beer Works</title>
+    <title>ENBW Beer List</title>
```

Comment:
_Previous review omitted this comment:_

This change will break the test suite, as seen here. We would likely want to update it accordingly.

https://github.com/ENBW/enbw/blob/a0dd616b44e5114ed82ab18e2fdea4fb373a85d7/test/controllers/static_controller_test.rb#L8